core/crypto/crypto_store/crypto_store_impl.cpp

```diff
@@ -96,29 +96,41 @@ namespace kagome::crypto {
   }
 
   CryptoStoreImpl::CryptoStoreImpl(
-      Path keys_directory,
       std::shared_ptr<ED25519Provider> ed25519_provider,
       std::shared_ptr<SR25519Provider> sr25519_provider,
       std::shared_ptr<Secp256k1Provider> secp256k1_provider,
       std::shared_ptr<Bip39Provider> bip39_provider,
       std::shared_ptr<CSPRNG> random_generator)
-      : keys_directory_(std::move(keys_directory)),
+      : keys_directory_(),
```

Comment:
If it has default ctor, we can pass it

Reply:
I already merged this branch into another (feature branch), where this moment is fixed, so I consider it would be counterproductive to make changes here.